roles/service-nginx-mitglieder/vars/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+
+mitglieder_php_obfuscator_password: "{{ lookup('password', '/dev/null length=20') }}"
```

Comment:
Das `mitglieder_php_obfuscator_password` ändert sich vermutlich mit jedem Aufruf der Rolle. Ist das beabsichtigt?

Reply:
Ja, das ist kein Problem. Kann nur in einem Fall eins werden: Jemand füllt gerade ein Formular aus, auf dem eine Eingabe falsch ist. In diesem Moment muss jemand die role ausrollen. Nicht gerade wahrscheinlich, kann also so bleiben :-)